src/plugins/intel_gna/docs/transformations.md

```diff
@@ -0,0 +1,97 @@
+# GNA transformations documentation
+
+GNAPlugin has multiple methods, one of the most important is LoadNetwork. That method prepares input graph to execute on the GNA hardware. Input graph usually consists of variety of operations, i.e. Convolution, Add, Gather, LSTMSequence. GNA hardware provides limited number of operations. First of all LoadNetwork method converts input network to equivalent network with layers each of them could be executed on the GNA hardware.
```

Comment:
GNA Plugin provides implementation of multiple methods required by OpenVINO plugin API. Input graph usually consists of variety of operations, i.e. Convolution, Add, Gather, LSTMSequence and so on. GNA Hardware is its own limitation and not all operations can be executed on GNA Hardware.
One of the main functionalities for GNA Plugin is conversion of input network to equivalent network which could be executed on the GNA hardware. This conversion is realized by LoadNetwok method.